pysb/generator/kappa.py

```diff
@@ -41,8 +51,9 @@ def generate_content(self):
     #            parent_name = ''
     #        else:
     #            parent_name = c.parent.name
-    #        self.__content += ("  %s  %d  %f  %s\n") % (c.name, c.dimension, c.size, parent_name)
-    #    self.__content += "end compartments\n\n"        
+    #        self.__content += ("  %s  %d  %f  %s\n") % \
```

Comment:
Unnecessary parens around the string here (although I see they were there before). Also pep8 says prefer parens over backslashes for line continuations. Could fix both issues at once like so:
```python
  self.__content += ("  %s ... " %
                     (c.name, ...))
```